spec/spec_helper_acceptance.rb

```diff
@@ -2,7 +2,7 @@
 require 'voxpupuli/acceptance/spec_helper_acceptance'
 
 configure_beaker do |_host|
-  install_module_from_forge('puppetlabs-apt', '>= 4.1.0 < 8.0.0') if fact('os.family') == 'Debian'
+  install_module_from_forge_on(host, 'puppetlabs-apt', '>= 4.1.0 < 8.0.0') if fact_on(host, 'os.family') == 'Debian'
```

Comment:
Note I suggested the on line, but you also need to change all others + rename `_host` to `host`.

Reply:
Fixed it.